src/iconsManifest/supportedFolders.ts

```diff
@@ -431,7 +435,7 @@ export const extensions: IFolderCollection = {
     },
     {
       icon: 'private',
-      extensions: ['private', '.private'],
+      extensions: ['private', '.private', 'dev', 'development'],
```

Comment:
Sorry but I don't agree with having `dev` or `development` under the private icon. This is a very opinionated decision and should follow the `customization` [approach](https://github.com/vscode-icons/vscode-icons/wiki/Customization).